core-bundle/contao/dca/tl_image_size.php

```diff
@@ -155,7 +155,8 @@
 			'inputType'               => 'checkboxWizard',
 			'options_callback'        => array('tl_image_size', 'getFormats'),
 			'eval'                    => array('multiple'=>true),
-			'sql'                     => "varchar(1024) NOT NULL default ''"
+			'sql'                     => "varchar(1024) NOT NULL default ''",
+			'relation'                => array('table'=>'tl_image_size', 'type'=>'hasMany', 'load'=>'lazy')
```

Comment:
This seems odd. The `getFormats()` method returns an array of strings and not a collection of image size models. Are you sure this is correct?

Reply:
you're right, that makes no sense.